public/locales/en/about.json

```diff
@@ -23,6 +23,12 @@
         },
         "part3": ". You could even host an air quality sensor in your backyard!"
     },
+    "sensorsDown": {
+        "heading": "Sensor Not Reporting AQI",
+        "part1": "There are many steps involved in calculating the AQI using the information reported by the Purple Air sensors that we host in South Gate. The steps of this algorithm have been outlined by the EPA, and we follow the EPA’s guidelines to ensure our sensors are accurate. ",
+        "part2": "Purple Air sensors are great for our work because they are affordable and relatively accurate given their cost. That being said, they are not as reliable as some more expensive sensors, which means that sometimes the values they record aren’t sufficient for calculating AQI. This could be very temporary (a matter of minutes or hours), or might last longer. If a sensor has been down for a long time, it probably means that either the sensor is no longer connected to the internet or that it has encountered a more serious hardware issue.",
```

Comment:
```suggestion
        "part2": "PurpleAir sensors are great for our work because they are affordable and relatively accurate given their cost. That being said, they are not as reliable as some more expensive sensors, which means that sometimes the values they record aren’t sufficient for calculating AQI. This could be very temporary (a matter of minutes or hours) or might last longer. If a sensor has been down for a long time, it probably means that either the sensor is no longer connected to the internet or that it has encountered a more serious hardware issue.",
```